EcommerceDataAnalysisToolServer/EcommerceDataAnalysisToolServer/Repository/SalesDataAnalysisRepository.cs

```diff
@@ -203,6 +203,17 @@ public async Task<List<Ecommerce>> GetSalesForMonthAndYearAsync(int month, int y
                                     .ToListAsync();
         }
 
+        /// <summary>
+        /// Method to get the total sale 
+        /// </summary>
+        /// <returns>return the total sale amount from the DB</returns>
+        public double GetTotalSales()
```

Comment:
We already have a method to handle total revenue in a year can you correct the code and raise PR again?

Reply:
This query fetches the sales record irrespective of the year (i.e. the total records sale) so have created a new one as the total revenue cannot be used.